knowledge-base/actions/atlassian/gajira-login/action-security.yml

```diff
@@ -0,0 +1,10 @@
+name: "Jira Login"
+github-token:
+  environment-variable-name: GITHUB_TOKEN
+  permissions:
```

Comment:
I think there is only `content: read` permission to retrieve diff in commit for todo, but there is no need for adding this permission as,
There is no need for adding permissions for KBs having only `content: read` permission, because it gets added at workflow level by default.
so please remove all the permissions for this KB.